init_venv.sh

```diff
@@ -1,10 +1,25 @@
 #!/usr/bin/env bash
+set -v
+set -x
 
 work_dir=$(realpath "$(dirname $0)")
 
 venv_dir=$1
+PYTHON_NAME=$2
+SC_SDK_REPO=$3
```

Comment:
I propose to not use this as a parameter of the script, but still take it as env var

Reply:
IMHO it is better to use explicit parameters of the script, instead of many-many implicit environment variables.
This is the idea `Explicit is better than implicit.` (yes, I think that it is a good idea to use [The Zen of Python](https://www.python.org/dev/peps/pep-0020/) in bash scripts 🙂 )

Particularly, when I used environment variables I had several times an issue when I had to open a new console, since environment variables in the previous console gave unknown side effects. It is not convenient

Concerning `PYTHON_NAME` -- in my desktop the default `python3` is `python3.6` (I do not know why), but the code does not work `python3.6` .
So, I think this parameter is required, but if you wish we could move it later (make it 4-th or 5-th parameter) and add the default variable.
And we could later add help to this script 🙂

Reply:
You can set this particular env inside ~/.bashrc for example. 
But passing this as a parameter makes me to explicitly pass a path and hard-coded version of python here https://github.com/openvinotoolkit/training_extensions/blob/brave_new_world/tools/instantiate.py#L95

Reply:
As soon as the our sdk is publicly available, both SC_SDK_REPO and SKIP_SC_SDK_ADDITIONAL_PACKAGES __parameters__ will have to be removed. So I think they should not be introduced.

Reply:
Regarding python version, I would add a check to this script to check that the version is >= 3.7

Reply:
>You can set this particular env inside ~/.bashrc for example.

Sorry, Ilya, I think that it is a bad idea. (Sometimes we make several experiments with slightly different options.)

>But passing this as a parameter makes me to explicitly pass a path and hard-coded version of python here https://github.com/openvinotoolkit/training_extensions/blob/brave_new_world/tools/instantiate.py#L95

Oh, I see. Yes, this is inconvenient.
Ok.
I will read these parameters as environment variables (unfortunately -- this is not a good decision, but it is better than change `instantiate.py`). 
And I will add check that `SC_SDK_REPO` environment variable is set.

>Regarding python version, I would add a check to this script to check that the version is >= 3.7
Ok.

Reply:
2 @Ilya-Krylov 
Ilya, I made the changes. Now the `SC_SDK_REPO` should be set as an environment variable.
Also I added a check the version of python is 3.7 or 3.8
Note that `PYTHON_NAME` has the default value `python3` (so if your default python is python3.8 the script will work without this parameter)